docs/install.md

```diff
@@ -3,26 +3,119 @@ id: install
 title: Install
 ---
 
-Install with `yarn`:
+First, install Prettier locally:
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--npm-->
 
 ```bash
-yarn add prettier --dev --exact
-# or globally
-yarn global add prettier
+npm install --save-dev --save-exact prettier
 ```
 
-_We're using `yarn` but you can use `npm` if you like:_
+<!--yarn-->
 
 ```bash
-npm install --save-dev --save-exact prettier
-# or globally
-npm install --global prettier
+yarn add --dev --exact prettier
+```
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+Then, create an empty config file to let editors and other tooling know you are using Prettier:
+
+```bash
+echo "{}" > .prettierrc.json
+```
+
+Next, create a [.prettierignore](ignore.md) file to let the Prettier CLI and editors know which files to _not_ format. Here’s an example:
+
 ```
+# Ignore artifacts:
+build
+coverage
+```
+
+> Tip! Base your .prettierignore on .gitignore and .eslintignore (if you have one).
+
+> Another tip! If your project isn’t ready to format, say, HTML files yet, add `*.html`.
 
-> We recommend pinning an exact version of prettier in your `package.json` as we introduce stylistic changes in patch releases.
+Now, format all files with Prettier:
 
-If you use `npx` to run Prettier, the version should be pinned like this:
+<!--DOCUSAURUS_CODE_TABS-->
+<!--npm-->
 
 ```bash
-npx prettier@2.0.5 . --write
+npx prettier --write .
 ```
+
+> What is that `npx` thing? `npx` ships with `npm` and lets you run locally installed tools. We’ll leave off the `npx` part for brevity throughout the rest of this file!
+>
+> Note: If you forget to install Prettier first, `npx` will temporarily download the latest version. That’s not good for Prettier! It’s important to have a locked down version of Prettier in your package.json. And it’s faster, too!
+
+<!--yarn-->
+
+```bash
+yarn prettier --write .
+```
+
+> What is `yarn` doing at the start? `yarn prettier` runs the locally installed version of Prettier. We’ll leave off the `yarn` part for brevity throughout the rest of this file!
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+`prettier --write .` is great for formatting everything, but for a big project it might take a little while. You may run `prettier --write app/` to format a certain directory, or `prettier --write app/components/Button.js` to format a certain files. Or `prettier --write "app/**/*.test.js"` to format all tests in a directory.
```

Comment:
```suggestion
`prettier --write .` is great for formatting everything, but for a big project it might take a little while. You may run `prettier --write app/` to format a certain directory, or `prettier --write app/components/Button.js` to format a certain file. Or use a more complex glob like `prettier --write "app/**/*.test.js"` to format all tests in a directory.
```

Explicitly saying this is a glob will help users search for docs on how to use globs more effectively since they’ll have a name to search for online.